### ISSUE TITLE
Adding this plugin to config causes data to be omitted from my webpack-stats output

### ISSUE THREAD

**Issue by grahamcracker** [getsentry/sentry-webpack-plugin#36]
This is a super bizarre bug, but I've investigated for hours and can't seem to find how the cause and effect here are related.

### Steps that result in the problem

1. My webpack setup uses webpack's CommonChunkPlugin twice, and the plugin section looks something like this:

```
plugins: [...,
new webpack.optimize.CommonsChunkPlugin({
      chunks: [ENTRY_MAIN_LOGGEDIN, ...Object.keys(ENTRY_PAGES_LOGGEDIN)],
      name: ENTRY_VENDOR_LOGGEDIN,
      minChunks: isNodeModule,
    }),
    new webpack.optimize.CommonsChunkPlugin({
      chunks: [ENTRY_MAIN, ...Object.keys(ENTRY_PAGES)],
      name: ENTRY_VENDOR,
      minChunks: isNodeModule,
    }), ...]
```

2. I install `sentry-webpack-plugin` directly as specified in the documentation. This includes adding to the plugins array:

```
new SentryCliPlugin({
      include: '.',
      ignoreFile: '.sentrycliignore',
      ignore: ['node_modules', 'webpack.config.js'],
      configFile: 'sentry.properties'
    })
```

3. I run `cross-env NODE_ENV='production' webpack --config webpack.config.js` to build my files.

### What happens

My output for webpack stats (webpack-stats.json) is drastically smaller than it should be. Specifically, it only contains two chunks (specifically with the names from constants ENTRY_VENDOR and ENTRY_MAIN). All of the other chunk locations (and all of the rest of my static files) are not included in webpack-stats.json.

The crazy part of this is that **if I remove the `new SentryCliPlugin()` block from my plugins array, my webpack stats file generates normally, with 20 chunks and a file about 6x longer**. The problem is directly reproducible by simply adding the SentryCliPlugin.

### What I've tried

- Checking for webpack errors in output (there were none)
- Reinstalling sentry-webpack-plugin into my project
- Checking github issues for both webpack and sentry-webpack-plugin for similar problems

Let me know if there is any more information I can provide here to get to the root of the problem! Thanks for the work on sentry-webpack-plugin.

**Comment by jan-auer** [comment]
Thank you for the detailed analysis of your problem. 👏  Just taking a wild guess here - could you try to move `sentry-webpack-plugin` **before** your common chunks plugins (or even any other plugins, actually)?

Regardless of the above, it would be interesting to see what the plugin does to the webpack compiler options in your case. Could you edit `node_modules/@sentry/webpack-plugin/src/index.js` around line *112* where it calls `injectRelease(compiler, versionPromise)` to something like below and then post the relevant console output?

```js
    console.log('\nBEFORE SENTRY PLUGIN');
    console.log('entry:', compiler.options.entry);
    console.log('rules:', compiler.options.module.rules);
    injectRelease(compiler, versionPromise);
    console.log('\nAFTER SENTRY PLUGIN');
    console.log('entry:', compiler.options.entry);
    console.log('rules:', compiler.options.module.rules);
    console.log();
```

**Comment by grahamcracker** [comment]
@jan-auer Thank you for the response!

I tried to move `sentry-webpack-plugin` before all the other plugins - this had no change in the problem.

Just tried the console logging you suggested, I'm not quite sure what I'm looking at here but let me know what you think:

```
BEFORE SENTRY PLUGIN
entry: () => Object.assign({}, entries)
rules: [ { test: /\.json$/,
    exclude: [ /node_modules/ ],
    loader: 'json-loader' },
  { test: /\.js$/,
    exclude: [ /scripts\/vendor/, /node_modules/ ],
    use: [ [Object] ] },
  { test: /\.(png|svg|jpg|gif)$/,
    exclude: [ /node_modules/ ],
    loader: 'url-loader?limit=15000&name=img/[name]-[hash:8].[ext]' },
  { test: /\.(woff|woff2|eot|ttf)$/,
    include: [ '/Users/graham/Documents/project/resources/fonts' ],
    loader: 'url-loader?limit=10000&name=fonts/[name].[ext]' },
  { test: /\.(scss|sass|css)$/,
    exclude: [ /style\/vendor/ ],
    use: [ [Object], [Object], [Object], [Object], [Object] ] } ]

AFTER SENTRY PLUGIN
entry: /Users/graham/Documents/project/node_modules/@sentry/webpack-plugin/src/sentry-webpack.module.js
rules: [ { test: /\.json$/,
    exclude: [ /node_modules/ ],
    loader: 'json-loader' },
  { test: /\.js$/,
    exclude: [ /scripts\/vendor/, /node_modules/ ],
    use: [ [Object] ] },
  { test: /\.(png|svg|jpg|gif)$/,
    exclude: [ /node_modules/ ],
    loader: 'url-loader?limit=15000&name=img/[name]-[hash:8].[ext]' },
  { test: /\.(woff|woff2|eot|ttf)$/,
    include: [ '/Users/graham/Documents/project/resources/fonts' ],
    loader: 'url-loader?limit=10000&name=fonts/[name].[ext]' },
  { test: /\.(scss|sass|css)$/,
    exclude: [ /style\/vendor/ ],
    use: [ [Object], [Object], [Object], [Object], [Object] ] },
  { test: /sentry-webpack\.module\.js$/, use: [ [Object] ] } ]
```

**Comment by culshaw** [comment]
I have the same issue.

I use a glob regex to generate a long entry array for my bundles (multiple).

I used the code suggested above to generate my before and after.

I added filtering to the entries to help sentry not try to process images etc.

sentry cli setup
```
new SentryCliPlugin({
    entries: /\.js$/, // <<<< This is the problem
    configFile: "sentry.properties",
    include: cwd + "/dist/js/"
})
```

before (aka without entry filtering):
```
entry: { 'images/admin/google_g_for_btn_red_32.png': './src/org/images/admin/google_g_for_btn_red_32.png',
  'images/animated-logo.gif': './src/org/images/animated-logo.gif',
  'images/benefits-suggestions.jpg': './src/org/images/benefits-suggestions.jpg',
  'images/bg.png': './src/org/images/bg.png',
  'images/common-triangle-arrow-down.svg': './src/org/images/common-triangle-arrow-down.svg',
  'images/dashboard/org_nyc_demo_profile.png': './src/org/images/dashboard/org_nyc_demo_profile.png',
  'images/direct_booking/testimonial-avatar.jpg': './src/org/images/direct_booking/testimonial-avatar.jpg',
  'images/email/Animated_tick_1.gif': './src/org/images/email/Animated_tick_1.gif',
  'images/email/Animated_tick.gif': './src/org/images/email/Animated_tick.gif',
  'images/email/arrow_down.png': './src/org/images/email/arrow_down.png',
  'images/email/circular_christian.png': './src/org/images/email/circular_christian.png',
  'images/email/circular_mark.png': './src/org/images/email/circular_mark.png',
  'images/email/circular_shelly.png': './src/org/images/email/circular_shelly.png',
  'images/email/circular_shib.png': './src/org/images/email/circular_shib.png',
  'images/email/Spacer.gif': './src/org/images/email/Spacer.gif',
  'images/email/org-Logo-Icon.gif': './src/org/images/email/org-Logo-Icon.gif',
  'images/email/org-Logo.gif': './src/org/images/email/org-Logo.gif',
  'images/email/org-Soc-LinkedIn.gif': './src/org/images/email/org-Soc-LinkedIn.gif',
  'images/email/org-Soc-Twitter.gif': './src/org/images/email/org-Soc-Twitter.gif',
  'images/email/orgboys.jpg': './src/org/images/email/orgboys.jpg',
  'images/favicons/amber/apple-touch-icon-114x114.png': './src/org/images/favicons/amber/apple-touch-icon-114x114.png',
  'images/favicons/amber/apple-touch-icon-120x120.png': './src/org/images/favicons/amber/apple-touch-icon-120x120.png',
  'images/favicons/amber/apple-touch-icon-144x144.png': './src/org/images/favicons/amber/apple-touch-icon-144x144.png',
  'images/favicons/amber/apple-touch-icon-152x152.png': './src/org/images/favicons/amber/apple-touch-icon-152x152.png',
  'images/favicons/amber/apple-touch-icon-57x57.png': './src/org/images/favicons/amber/apple-touch-icon-57x57.png',
  'images/favicons/amber/apple-touch-icon-60x60.png': './src/org/images/favicons/amber/apple-touch-icon-60x60.png',
  'images/favicons/amber/apple-touch-icon-72x72.png': './src/org/images/favicons/amber/apple-touch-icon-72x72.png',
  'images/favicons/amber/apple-touch-icon-76x76.png': './src/org/images/favicons/amber/apple-touch-icon-76x76.png',
  'images/favicons/amber/favicon-128x128.png': './src/org/images/favicons/amber/favicon-128x128.png',
  'images/favicons/amber/favicon-16x16.png': './src/org/images/favicons/amber/favicon-16x16.png',
  'images/favicons/amber/favicon-196x196.png': './src/org/images/favicons/amber/favicon-196x196.png',
  'images/favicons/amber/favicon-32x32.png': './src/org/images/favicons/amber/favicon-32x32.png',
  'images/favicons/amber/favicon-96x96.png': './src/org/images/favicons/amber/favicon-96x96.png',
  'images/favicons/amber/mstile-144x144.png': './src/org/images/favicons/amber/mstile-144x144.png',
  'images/favicons/amber/mstile-150x150.png': './src/org/images/favicons/amber/mstile-150x150.png',
  'images/favicons/amber/mstile-310x150.png': './src/org/images/favicons/amber/mstile-310x150.png',
  'images/favicons/amber/mstile-310x310.png': './src/org/images/favicons/amber/mstile-310x310.png',
  'images/favicons/amber/mstile-70x70.png': './src/org/images/favicons/amber/mstile-70x70.png',
  'images/favicons/black/apple-touch-icon-114x114.png': './src/org/images/favicons/black/apple-touch-icon-114x114.png',
  'images/favicons/black/apple-touch-icon-120x120.png': './src/org/images/favicons/black/apple-touch-icon-120x120.png',
  'images/favicons/black/apple-touch-icon-144x144.png': './src/org/images/favicons/black/apple-touch-icon-144x144.png',
  'images/favicons/black/apple-touch-icon-152x152.png': './src/org/images/favicons/black/apple-touch-icon-152x152.png',
  'images/favicons/black/apple-touch-icon-57x57.png': './src/org/images/favicons/black/apple-touch-icon-57x57.png',
  'images/favicons/black/apple-touch-icon-60x60.png': './src/org/images/favicons/black/apple-touch-icon-60x60.png',
  'images/favicons/black/apple-touch-icon-72x72.png': './src/org/images/favicons/black/apple-touch-icon-72x72.png',
  'images/favicons/black/apple-touch-icon-76x76.png': './src/org/images/favicons/black/apple-touch-icon-76x76.png',
  'images/favicons/black/favicon-128x128.png': './src/org/images/favicons/black/favicon-128x128.png',
  'images/favicons/black/favicon-144x144.png': './src/org/images/favicons/black/favicon-144x144.png',
  'images/favicons/black/favicon-152x152.png': './src/org/images/favicons/black/favicon-152x152.png',
  'images/favicons/black/favicon-160x160.png': './src/org/images/favicons/black/favicon-160x160.png',
  'images/favicons/black/favicon-16x16.png': './src/org/images/favicons/black/favicon-16x16.png',
  'images/favicons/black/favicon-192x192.png': './src/org/images/favicons/black/favicon-192x192.png',
  'images/favicons/black/favicon-196x196.png': './src/org/images/favicons/black/favicon-196x196.png',
  'images/favicons/black/favicon-32x32.png': './src/org/images/favicons/black/favicon-32x32.png',
  'images/favicons/black/favicon-384x384.png': './src/org/images/favicons/black/favicon-384x384.png',
  'images/favicons/black/favicon-512x512.png': './src/org/images/favicons/black/favicon-512x512.png',
  'images/favicons/black/favicon-72x72.png': './src/org/images/favicons/black/favicon-72x72.png',
  'images/favicons/black/favicon-96x96.png': './src/org/images/favicons/black/favicon-96x96.png',
  'images/favicons/black/mstile-144x144.png': './src/org/images/favicons/black/mstile-144x144.png',
  'images/favicons/black/mstile-150x150.png': './src/org/images/favicons/black/mstile-150x150.png',
  'images/favicons/black/mstile-310x310.png': './src/org/images/favicons/black/mstile-310x310.png',
  'images/favicons/black/mstile-70x70.png': './src/org/images/favicons/black/mstile-70x70.png',
  'images/favicons/green/apple-touch-icon-114x114.png': './src/org/images/favicons/green/apple-touch-icon-114x114.png',
  'images/favicons/green/apple-touch-icon-120x120.png': './src/org/images/favicons/green/apple-touch-icon-120x120.png',
  'images/favicons/green/apple-touch-icon-144x144.png': './src/org/images/favicons/green/apple-touch-icon-144x144.png',
  'images/favicons/green/apple-touch-icon-152x152.png': './src/org/images/favicons/green/apple-touch-icon-152x152.png',
  'images/favicons/green/apple-touch-icon-57x57.png': './src/org/images/favicons/green/apple-touch-icon-57x57.png',
  'images/favicons/green/apple-touch-icon-60x60.png': './src/org/images/favicons/green/apple-touch-icon-60x60.png',
  'images/favicons/green/apple-touch-icon-72x72.png': './src/org/images/favicons/green/apple-touch-icon-72x72.png',
  'images/favicons/green/apple-touch-icon-76x76.png': './src/org/images/favicons/green/apple-touch-icon-76x76.png',
  'images/favicons/green/favicon-128x128.png': './src/org/images/favicons/green/favicon-128x128.png',
  'images/favicons/green/favicon-16x16.png': './src/org/images/favicons/green/favicon-16x16.png',
  'images/favicons/green/favicon-196x196.png': './src/org/images/favicons/green/favicon-196x196.png',
  'images/favicons/green/favicon-32x32.png': './src/org/images/favicons/green/favicon-32x32.png',
  'images/favicons/green/favicon-96x96.png': './src/org/images/favicons/green/favicon-96x96.png',
  'images/favicons/green/mstile-144x144.png': './src/org/images/favicons/green/mstile-144x144.png',
  'images/favicons/green/mstile-150x150.png': './src/org/images/favicons/green/mstile-150x150.png',
  'images/favicons/green/mstile-310x310.png': './src/org/images/favicons/green/mstile-310x310.png',
  'images/favicons/green/mstile-70x70.png': './src/org/images/favicons/green/mstile-70x70.png',
  'images/favicons/pink/apple-touch-icon-114x114.png': './src/org/images/favicons/pink/apple-touch-icon-114x114.png',
  'images/favicons/pink/apple-touch-icon-120x120.png': './src/org/images/favicons/pink/apple-touch-icon-120x120.png',
  'images/favicons/pink/apple-touch-icon-144x144.png': './src/org/images/favicons/pink/apple-touch-icon-144x144.png',
  'images/favicons/pink/apple-touch-icon-152x152.png': './src/org/images/favicons/pink/apple-touch-icon-152x152.png',
  'images/favicons/pink/apple-touch-icon-57x57.png': './src/org/images/favicons/pink/apple-touch-icon-57x57.png',
  'images/favicons/pink/apple-touch-icon-60x60.png': './src/org/images/favicons/pink/apple-touch-icon-60x60.png',
  'images/favicons/pink/apple-touch-icon-72x72.png': './src/org/images/favicons/pink/apple-touch-icon-72x72.png',
  'images/favicons/pink/apple-touch-icon-76x76.png': './src/org/images/favicons/pink/apple-touch-icon-76x76.png',
  'images/favicons/pink/favicon-128x128.png': './src/org/images/favicons/pink/favicon-128x128.png',
  'images/favicons/pink/favicon-16x16.png': './src/org/images/favicons/pink/favicon-16x16.png',
  'images/favicons/pink/favicon-196x196.png': './src/org/images/favicons/pink/favicon-196x196.png',
  'images/favicons/pink/favicon-32x32.png': './src/org/images/favicons/pink/favicon-32x32.png',
  'images/favicons/pink/favicon-96x96.png': './src/org/images/favicons/pink/favicon-96x96.png',
  'images/favicons/pink/mstile-144x144.png': './src/org/images/favicons/pink/mstile-144x144.png',
  'images/favicons/pink/mstile-150x150.png': './src/org/images/favicons/pink/mstile-150x150.png',
  'images/favicons/pink/mstile-310x310.png': './src/org/images/favicons/pink/mstile-310x310.png',
  'images/favicons/pink/mstile-70x70.png': './src/org/images/favicons/pink/mstile-70x70.png',
  'images/favicons/red/apple-touch-icon-114x114.png': './src/org/images/favicons/red/apple-touch-icon-114x114.png',
  'images/favicons/red/apple-touch-icon-120x120.png': './src/org/images/favicons/red/apple-touch-icon-120x120.png',
  'images/favicons/red/apple-touch-icon-144x144.png': './src/org/images/favicons/red/apple-touch-icon-144x144.png',
  'images/favicons/red/apple-touch-icon-152x152.png': './src/org/images/favicons/red/apple-touch-icon-152x152.png',
  'images/favicons/red/apple-touch-icon-57x57.png': './src/org/images/favicons/red/apple-touch-icon-57x57.png',
  'images/favicons/red/apple-touch-icon-60x60.png': './src/org/images/favicons/red/apple-touch-icon-60x60.png',
  'images/favicons/red/apple-touch-icon-72x72.png': './src/org/images/favicons/red/apple-touch-icon-72x72.png',
  'images/favicons/red/apple-touch-icon-76x76.png': './src/org/images/favicons/red/apple-touch-icon-76x76.png',
  'images/favicons/red/favicon-128x128.png': './src/org/images/favicons/red/favicon-128x128.png',
  'images/favicons/red/favicon-16x16.png': './src/org/images/favicons/red/favicon-16x16.png',
  'images/favicons/red/favicon-196x196.png': './src/org/images/favicons/red/favicon-196x196.png',
  'images/favicons/red/favicon-32x32.png': './src/org/images/favicons/red/favicon-32x32.png',
  'images/favicons/red/favicon-96x96.png': './src/org/images/favicons/red/favicon-96x96.png',
  'images/favicons/red/mstile-144x144.png': './src/org/images/favicons/red/mstile-144x144.png',
  'images/favicons/red/mstile-150x150.png': './src/org/images/favicons/red/mstile-150x150.png',
  'images/favicons/red/mstile-310x310.png': './src/org/images/favicons/red/mstile-310x310.png',
  'images/favicons/red/mstile-70x70.png': './src/org/images/favicons/red/mstile-70x70.png',
  'images/homepage/akqa_avatar.png': './src/org/images/homepage/akqa_avatar.png',
  'images/homepage/client/adamandeveDDB.png': './src/org/images/homepage/client/adamandeveDDB.png',
  'images/homepage/client/akqa.png': './src/org/images/homepage/client/akqa.png',
  'images/homepage/client/bbh.png': './src/org/images/homepage/client/bbh.png',
  'images/homepage/client/moo.png': './src/org/images/homepage/client/moo.png',
  'images/homepage/client/saatchi.png': './src/org/images/homepage/client/saatchi.png',
  'images/homepage/client/sapient.png': './src/org/images/homepage/client/sapient.png',
  'images/homepage/client/ustwo.png': './src/org/images/homepage/client/ustwo.png',
  'images/homepage/client/virgin.png': './src/org/images/homepage/client/virgin.png',
  'images/homepage/desktop/hero.jpg': './src/org/images/homepage/desktop/hero.jpg',
  'images/homepage/desktop/hero.png': './src/org/images/homepage/desktop/hero.png',
  'images/homepage/freelancer/client-service.png': './src/org/images/homepage/freelancer/client-service.png',
  'images/homepage/freelancer/creative.png': './src/org/images/homepage/freelancer/creative.png',
  'images/homepage/freelancer/delivery.png': './src/org/images/homepage/freelancer/delivery.png',
  'images/homepage/freelancer/designer.png': './src/org/images/homepage/freelancer/designer.png',
  'images/homepage/freelancer/developer.png': './src/org/images/homepage/freelancer/developer.png',
  'images/homepage/freelancer/social.png': './src/org/images/homepage/freelancer/social.png',
  'images/homepage/freelancer/ux.png': './src/org/images/homepage/freelancer/ux.png',
  'images/homepage/hexagon-tick.png': './src/org/images/homepage/hexagon-tick.png',
  'images/homepage/hexagon-tick.svg': './src/org/images/homepage/hexagon-tick.svg',
  'images/homepage/orgboys-lg.jpg': './src/org/images/homepage/orgboys-lg.jpg',
  'images/homepage/orgboys.jpg': './src/org/images/homepage/orgboys.jpg',
  'images/icons/_social-black.svg': './src/org/images/icons/_social-black.svg',
  'images/icons/adamandeveDDB_logo.svg': './src/org/images/icons/adamandeveDDB_logo.svg',
  'images/icons/AKQA_Logo.svg': './src/org/images/icons/AKQA_Logo.svg',
  'images/icons/arrow-down-black.svg': './src/org/images/icons/arrow-down-black.svg',
  'images/icons/arrow-down-var-grey.svg': './src/org/images/icons/arrow-down-var-grey.svg',
  'images/icons/arrow-down-white.svg': './src/org/images/icons/arrow-down-white.svg',
  'images/icons/arrow-right-coral.svg': './src/org/images/icons/arrow-right-coral.svg',
  'images/icons/bbh_logo.svg': './src/org/images/icons/bbh_logo.svg',
  'images/icons/blank-page-logo.svg': './src/org/images/icons/blank-page-logo.svg',
  'images/icons/booking.svg': './src/org/images/icons/booking.svg',
  'images/icons/caret-black.svg': './src/org/images/icons/caret-black.svg',
  'images/icons/caret-var-grey-darker-left.svg': './src/org/images/icons/caret-var-grey-darker-left.svg',
  'images/icons/caret-var-grey-darker-right.svg': './src/org/images/icons/caret-var-grey-darker-right.svg',
  'images/icons/caret-white.svg': './src/org/images/icons/caret-white.svg',
  'images/icons/cat_client_services.svg': './src/org/images/icons/cat_client_services.svg',
  'images/icons/cat_creative.svg': './src/org/images/icons/cat_creative.svg',
  'images/icons/cat_delivery.svg': './src/org/images/icons/cat_delivery.svg',
  'images/icons/cat_design.svg': './src/org/images/icons/cat_design.svg',
  'images/icons/cat_developer.svg': './src/org/images/icons/cat_developer.svg',
  'images/icons/cat_social.svg': './src/org/images/icons/cat_social.svg',
  'images/icons/cat_testing.svg': './src/org/images/icons/cat_testing.svg',
  'images/icons/cat_ux.svg': './src/org/images/icons/cat_ux.svg',
  'images/icons/clipboarder.png': './src/org/images/icons/clipboarder.png',
  'images/icons/conversations_red.svg': './src/org/images/icons/conversations_red.svg',
  'images/icons/conversations.svg': './src/org/images/icons/conversations.svg',
  'images/icons/cross-black.svg': './src/org/images/icons/cross-black.svg',
  'images/icons/cross-blue.png': './src/org/images/icons/cross-blue.png',
  'images/icons/cross-coral.svg': './src/org/images/icons/cross-coral.svg',
  'images/icons/cross-var-red.svg': './src/org/images/icons/cross-var-red.svg',
  'images/icons/cross-white.svg': './src/org/images/icons/cross-white.svg',
  'images/icons/email/envelope.png': './src/org/images/icons/email/envelope.png',
  'images/icons/email/linkedin.png': './src/org/images/icons/email/linkedin.png',
  'images/icons/email/twitter.png': './src/org/images/icons/email/twitter.png',
  'images/icons/exclam-circled-black.svg': './src/org/images/icons/exclam-circled-black.svg',
  'images/icons/exclam-circled-var-red.svg': './src/org/images/icons/exclam-circled-var-red.svg',
  'images/icons/exclam-circled-white.svg': './src/org/images/icons/exclam-circled-white.svg',
  'images/icons/file-white.svg': './src/org/images/icons/file-white.svg',
  'images/icons/hexagon-tick.svg': './src/org/images/icons/hexagon-tick.svg',
  'images/icons/homepage/client_services.svg': './src/org/images/icons/homepage/client_services.svg',
  'images/icons/homepage/creative.svg': './src/org/images/icons/homepage/creative.svg',
  'images/icons/homepage/delivery.svg': './src/org/images/icons/homepage/delivery.svg',
  'images/icons/homepage/design.svg': './src/org/images/icons/homepage/design.svg',
  'images/icons/homepage/developer.svg': './src/org/images/icons/homepage/developer.svg',
  'images/icons/homepage/social.svg': './src/org/images/icons/homepage/social.svg',
  'images/icons/homepage/ux.svg': './src/org/images/icons/homepage/ux.svg',
  'images/icons/linkedin_32.png': './src/org/images/icons/linkedin_32.png',
  'images/icons/lock-white.svg': './src/org/images/icons/lock-white.svg',
  'images/icons/lock.svg': './src/org/images/icons/lock.svg',
  'images/icons/logo_2017_white_shaded.svg': './src/org/images/icons/logo_2017_white_shaded.svg',
  'images/icons/marketplace-search-icon.svg': './src/org/images/icons/marketplace-search-icon.svg',
  'images/icons/moo_logo.svg': './src/org/images/icons/moo_logo.svg',
  'images/icons/note-locked.svg': './src/org/images/icons/note-locked.svg',
  'images/icons/notes.svg': './src/org/images/icons/notes.svg',
  'images/icons/open_envelope_amber.svg': './src/org/images/icons/open_envelope_amber.svg',
  'images/icons/open_envelope_coral.svg': './src/org/images/icons/open_envelope_coral.svg',
  'images/icons/open_envelope_green.svg': './src/org/images/icons/open_envelope_green.svg',
  'images/icons/open_envelope_grey.svg': './src/org/images/icons/open_envelope_grey.svg',
  'images/icons/open_envelope_red.svg': './src/org/images/icons/open_envelope_red.svg',
  'images/icons/progress_complete.png': './src/org/images/icons/progress_complete.png',
  'images/icons/required-triangle.svg': './src/org/images/icons/required-triangle.svg',
  'images/icons/right-angle-triangle-var-grey-light.svg': './src/org/images/icons/right-angle-triangle-var-grey-light.svg',
  'images/icons/right-angle-triangle-var-grey.svg': './src/org/images/icons/right-angle-triangle-var-grey.svg',
  'images/icons/right-angle-triangle-var-white.svg': './src/org/images/icons/right-angle-triangle-var-white.svg',
  'images/icons/role-expand.svg': './src/org/images/icons/role-expand.svg',
  'images/icons/saatchi_logo.svg': './src/org/images/icons/saatchi_logo.svg',
  'images/icons/SapientNitro_logo.svg': './src/org/images/icons/SapientNitro_logo.svg',
  'images/icons/silhouette.png': './src/org/images/icons/silhouette.png',
  'images/icons/social-black.svg': './src/org/images/icons/social-black.svg',
  'images/icons/social-white.svg': './src/org/images/icons/social-white.svg',
  'images/icons/star.svg': './src/org/images/icons/star.svg',
  'images/icons/tick_cross.png': './src/org/images/icons/tick_cross.png',
  'images/icons/tick_homepage.svg': './src/org/images/icons/tick_homepage.svg',
  'images/icons/tick-black.svg': './src/org/images/icons/tick-black.svg',
  'images/icons/tick-var-green.svg': './src/org/images/icons/tick-var-green.svg',
  'images/icons/tick-white.svg': './src/org/images/icons/tick-white.svg',
  'images/icons/trash_coral.svg': './src/org/images/icons/trash_coral.svg',
  'images/icons/trash.svg': './src/org/images/icons/trash.svg',
  'images/icons/twitter_32.png': './src/org/images/icons/twitter_32.png',
  'images/icons/undo-black.svg': './src/org/images/icons/undo-black.svg',
  'images/icons/undo-white.svg': './src/org/images/icons/undo-white.svg',
  'images/icons/untrash_coral.svg': './src/org/images/icons/untrash_coral.svg',
  'images/icons/usTwo_logo.svg': './src/org/images/icons/usTwo_logo.svg',
  'images/icons/verified.png': './src/org/images/icons/verified.png',
  'images/icons/virgin_logo.svg': './src/org/images/icons/virgin_logo.svg',
  'images/icons/white-triangle-reverse.svg': './src/org/images/icons/white-triangle-reverse.svg',
  'images/icons/work_history_verified.png': './src/org/images/icons/work_history_verified.png',
  'images/icons/work_history_org_verified.png': './src/org/images/icons/work_history_org_verified.png',
  'images/icons/org_logo_2015_coral.svg': './src/org/images/icons/org_logo_2015_coral.svg',
  'images/icons/org_logo_2015_wo.svg': './src/org/images/icons/org_logo_2015_wo.svg',
  'images/icons/org_logo_2015.svg': './src/org/images/icons/org_logo_2015.svg',
  'images/icons/org_logo-full-2017.svg': './src/org/images/icons/org_logo-full-2017.svg',
  'images/icons/org-logo-black.svg': './src/org/images/icons/org-logo-black.svg',
  'images/icons/org-logo-coral.svg': './src/org/images/icons/org-logo-coral.svg',
  'images/icons/org-logo-grey-for-emails--wide.png': './src/org/images/icons/org-logo-grey-for-emails--wide.png',
  'images/icons/org-logo-grey-for-emails.png': './src/org/images/icons/org-logo-grey-for-emails.png',
  'images/icons/org-logo-white.svg': './src/org/images/icons/org-logo-white.svg',
  'images/landing_page/403.jpg': './src/org/images/landing_page/403.jpg',
  'images/landing_page/404.jpg': './src/org/images/landing_page/404.jpg',
  'images/landing_page/500.png': './src/org/images/landing_page/500.png',
  'images/landing_page/expired_error.png': './src/org/images/landing_page/expired_error.png',
  'images/mappin.svg': './src/org/images/mappin.svg',
  'images/powered_by_onfido_@2x.png': './src/org/images/powered_by_onfido_@2x.png',
  'images/profiles/01_city_image--small.jpg': './src/org/images/profiles/01_city_image--small.jpg',
  'images/profiles/01_city_image.jpg': './src/org/images/profiles/01_city_image.jpg',
  'images/profiles/london.jpg': './src/org/images/profiles/london.jpg',
  'images/profiles/new_york.jpg': './src/org/images/profiles/new_york.jpg',
  'images/static_pages/403.jpg': './src/org/images/static_pages/403.jpg',
  'images/static_pages/404.jpg': './src/org/images/static_pages/404.jpg',
  'images/static_pages/biff_tannen.jpg': './src/org/images/static_pages/biff_tannen.jpg',
  'images/static_pages/doc_brown.jpg': './src/org/images/static_pages/doc_brown.jpg',
  'images/static_pages/fox.jpg': './src/org/images/static_pages/fox.jpg',
  'images/static_pages/gandalf.jpg': './src/org/images/static_pages/gandalf.jpg',
  'images/static_pages/george_mcfly.jpg': './src/org/images/static_pages/george_mcfly.jpg',
  'images/static_pages/jcvd.jpg': './src/org/images/static_pages/jcvd.jpg',
  'images/static_pages/lorraine.jpg': './src/org/images/static_pages/lorraine.jpg',
  'images/static_pages/maintenance.gif': './src/org/images/static_pages/maintenance.gif',
  'images/static_pages/old_biff_tannen.jpg': './src/org/images/static_pages/old_biff_tannen.jpg',
  'images/static_pages/owen.png': './src/org/images/static_pages/owen.png',
  'images/static_pages/willem.png': './src/org/images/static_pages/willem.png',
  'org/js/Addresses': './src/org/js/Apps/Addresses/index.js',
  'org/js/BlackBook': './src/org/js/Apps/BlackBook/index.js',
  'org/js/Briefs': './src/org/js/Apps/Briefs/index.js',
  'org/js/BriefsStandaloneShortlist': './src/org/js/Apps/BriefsStandaloneShortlist/index.js',
  'org/js/CompleteContractForm': './src/org/js/Apps/CompleteContractForm/index.js',
  'org/js/Contract': './src/org/js/Apps/Contract/index.js',
  'org/js/Core': './src/org/js/Apps/Core/index.js',
  'org/js/EnableOvertimeForm': './src/org/js/Apps/EnableOvertimeForm/index.js',
  'org/js/FreelancerInvoices': './src/org/js/Apps/FreelancerInvoices/index.js',
  'org/js/FreelancerProfile': './src/org/js/Apps/FreelancerProfile/index.js',
  'org/js/FreelancerTeam': './src/org/js/Apps/FreelancerTeam/index.js',
  'org/js/Geocomplete': './src/org/js/Apps/Geocomplete/index.js',
  'org/js/Invoices': './src/org/js/Apps/Invoices/index.js',
  'org/js/PaymentStatus': './src/org/js/Apps/PaymentStatus/index.js',
  'org/js/References': './src/org/js/Apps/References/index.js',
  'org/js/WorkHistory': './src/org/js/Apps/WorkHistory/index.js',
  'org/js/core.components.js': './src/org/js/Legacy/Root/core.components.js',
  'css/availability-badge.scss': './src/org/scss/availability-badge.scss',
  'css/core.scss': './src/org/scss/core.scss',
  'css/print.scss': './src/org/scss/print.scss' }
rules: [ { test: /\.js$/,
    include: /(Apps|Libs|Legacy\/Root|Legacy\/Components)/,
    exclude: /node_modules/,
    use: [ 'babel-loader', 'eslint-loader' ] },
  { test: /(core|availability-badge)\.scss$/,
    use: [ [Object], [Object], [Object], [Object] ],
    include: 
     [ '/Users/company/website/company/frontend',
       '/Users/company/website/company/frontend/node_modules' ] },
  { test: /print\.scss$/,
    use: [ [Object], [Object], [Object], [Object] ],
    include: 
     [ '/Users/company/website/company/frontend',
       '/Users/company/website/company/frontend/node_modules' ] },
  { test: /\.(jpe?g|gif|png|svg|woff|ttf|wav|mp3)$/,
    use: { loader: 'file-loader', options: [Object] } } ]
```

after:
```
entry: { 'org/js/core.components.js': 
   [ '/Users/company/website/node_modules/@sentry/webpack-plugin/src/sentry-webpack.module.js',
     './src/org/js/Legacy/Root/core.components.js' ] }
rules: [ { test: /\.js$/,
    include: /(Apps|Libs|Legacy\/Root|Legacy\/Components)/,
    exclude: /node_modules/,
    use: [ 'babel-loader', 'eslint-loader' ] },
  { test: /(core|availability-badge)\.scss$/,
    use: [ [Object], [Object], [Object], [Object] ],
    include: 
     [ '/Users/company/website/company/frontend',
       '/Users/company/website/company/frontend/node_modules' ] },
  { test: /print\.scss$/,
    use: [ [Object], [Object], [Object], [Object] ],
    include: 
     [ '/Users/company/website/company/frontend',
       '/Users/company/website/company/frontend/node_modules' ] },
  { test: /\.(jpe?g|gif|png|svg|woff|ttf|wav|mp3)$/,
    use: { loader: 'file-loader', options: [Object] } },
  { test: /sentry-webpack\.module\.js$/, use: [ [Object] ] } ]
```

**Comment by jan-auer** [comment]
@kamilogorek I was thinking we could add debug output like the above by default for the user to activate via environment variables maybe?

**Comment by kamilogorek** [comment]
@grahamcracker @culshaw released as `1.5.2`
